src/components/_internal/BaseDateRangePicker/BaseDateRangePicker.tsx

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import DatePicker from 'react-datepicker';
 import { any, prop } from 'ramda';
 import { isNotNull } from 'ramda-adjunct';
 
 import { FlexContainer } from '../../FlexContainer';
-import { datePickerStyles, dateRangePickerStyles } from './styles';
+import { dateRangePickerStyles } from './styles';
 import {
   BaseDateRangePickerPropTypes,
   BaseDateRangePickerProps,
   BaseDateRangePlaceholderPropTypes,
 } from './BaseDateRangePicker.types';
+import { StyledDatePicker as SingleStyledDatePicker } from '../BaseSingleDatePicker/SingleDatePicker';
+import { BaseSingleDatePicker } from '../BaseSingleDatePicker';
+import { pxToRem } from '../../../utils';
 
 const StyledDatePicker = styled(FlexContainer)`
```

Comment:
We can change this into:
```js
const StyledDatePicker = styled.div`
  ${dateRangePickerStyles}
  width: 100%;
`;
```
and then wrap both `BaseSingleDatePicker` in the `Inline` like:
```jsx
<StyledDatePicker>
  <Inline gap={SpaceSizes.sm} stretch={InlineEnums.StretchEnum.all}>
     <BaseSingleDatePicker ... />
     <BaseSingleDatePicker ... />